internal/parser/node_groups.go

```diff
@@ -17,7 +17,7 @@ type inputNodeGroup struct {
 	MovementModel      any `default:"Static"`
 	NodesType          any `default:"PC"`
 	Networks           []string
-	PredefinedPosition bool
+	PredefinedPosition any `default:"true"`
```

Comment:
This approach doesn't resolve my annotation from here: https://github.com/stg-tud/bp2022_netlab/pull/66#discussion_r1141464016. Still, the user would have to specify a position _and_ set the bool to `true`. From my perspective, setting the position should be enough.

Reply:
Also: What happens, if the user sets only the `X` or only the `Y` part of the position?